src/options.html

```diff
@@ -59,6 +59,24 @@ <h3 data-i18n-message-id="tabBehavior"></h3>
         <p><em data-i18n-message-id="replaceTabDescription"></em></p>
       </div>
 
+      <!--
+        TODO
+        - Add data-i18n
+      -->
+      <h3 data-i18n-message-id="theme"></h3>
```

Comment:
Looks like we still need to add all these messages to the https://github.com/mozilla-l10n/multi-account-containers-l10n/search?q=theme repo? Are you comfortable doing that? Or maybe @maxxcrawford can do that part?

Reply:
See https://github.com/mozilla-l10n/multi-account-containers-l10n/pull/12